app/api/order_statistics/events.py

```diff
@@ -33,25 +33,55 @@ class Meta:
-            Order.event_id == obj_id, Order.status == 'cancelled').scalar()
-        pending = db.session.query(func.sum(OrderTicket.quantity.label('sum'))).join(Order.order_tickets).filter(
-            Order.event_id == obj_id, Order.status == 'pending').scalar()
-        expired = db.session.query(func.sum(OrderTicket.quantity.label('sum'))).join(Order.order_tickets).filter(
-            Order.event_id == obj_id, Order.status == 'expired').scalar()
-        placed = db.session.query(func.sum(OrderTicket.quantity.label('sum'))).join(Order.order_tickets).filter(
-            Order.event_id == obj_id, Order.status == 'placed').scalar()
-        completed = db.session.query(func.sum(OrderTicket.quantity.label('sum'))).join(Order.order_tickets).filter(
-            Order.event_id == obj_id, Order.status == 'completed').scalar()
-        max = db.session.query(func.sum(Ticket.quantity.label('sum'))).filter(Ticket.event_id == obj_id,
-                                                                              Ticket.deleted_at == None).scalar()
+        total = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id)
+            .scalar()
+        )
+        draft = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id, Order.status == 'draft')
+            .scalar()
+        )
+        cancelled = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id, Order.status == 'cancelled')
+            .scalar()
+        )
+        pending = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id, Order.status == 'pending')
+            .scalar()
+        )
+        expired = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id, Order.status == 'expired')
+            .scalar()
+        )
+        placed = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id, Order.status == 'placed')
+            .scalar()
+        )
+        completed = (
+            db.session.query(func.sum(OrderTicket.quantity.label('sum')))
+            .join(Order.order_tickets)
+            .filter(Order.event_id == obj_id, Order.status == 'completed')
+            .scalar()
+        )
+        max = (
+            db.session.query(func.sum(Ticket.quantity.label('sum')))
+            .filter(Ticket.event_id == obj_id, Ticket.deleted_at == None)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4884175)